### DISCUSSION TITLE
我想问下探索结束会返回庭院不动的问题

### DISCUSSION THREAD

**Discussion by Eiko920** [lisai9093/YYS#19] [Q&A]
在探索boss打完返回到探索地图是发现了宝箱奖励后，会点击返回回到庭院然后识别不到困二八而停在庭院是什么问题

**Comment by lisai9093** [comment]
是单人探索模式么？那应该是不同庭院皮肤导致的“探索”图标不一致。
你试试修改下png文件夹下的"ditu.png"图片，比如现在默认的是：
<img width="45" alt="image" src="https://github.com/user-attachments/assets/3efdbf79-6021-4917-b54c-7ecfe12f8c27">
